spec/catalog.yaml

```diff
@@ -278,6 +339,21 @@ components:
             "instance": "/v1/namespaces"
           }
 
+    NotFoundResponse:
```

Comment:
nit: can we order the errors by error code, and place this after ForbiddenResponse?

Reply:
yes